{"name": "test_03_verify_contactUs", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <test_profile_elements.TestBestInTownPizzaProfile testMethod=test_03_verify_contactUs>\n\n    def setUp(self):\n    \n        # chrome_options = Options()\n        # chrome_options.add_argument(\"--headless\")\n        # self.driver = webdriver.Chrome(executable_path=\"C:/Users/Administrator/Desktop/BEST_IN_TOWN/chromedriver.exe\", options=chrome_options)\n    \n        self.driver = webdriver.Chrome(\n            \"C:/Users/Digital Suppliers/Desktop/bestintown/chromedriver.exe\")\n    \n        f = open('data.json', \"r\")\n        data = json.loads(f.read())\n>       fetching_website(self.driver, data[\"url\"])\n\ntest_profile_elements.py:33: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nmethods.py:34: in fetching_website\n    raise e\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0882a240a9162ef32ec47129a552d96d\")>, site_url = 'http://bestintownpizzas.com/home'\n\n    @allure.step(\"To fetch the website with url: {1}\")\n    def fetching_website(driver, site_url):\n        try:\n            driver.get(site_url)\n            # driver.maximize_window()\n            driver.set_window_size(1536, 864)\n            driver.implicitly_wait(5)\n            print(driver.title)\n            allure.attach(driver.get_screenshot_as_png(),\n                          name='website_screen', attachment_type=AttachmentType.PNG)\n            try:\n                driver.find_element_by_id(loginlocators.start_button).click()\n            except:\n                print(\"Restuarant has opened now\")\n            time.sleep(9)\n>           assert driver.title == \"Home | Best in Town Pizza\"\nE           AssertionError\n\nmethods.py:31: AssertionError"}, "description": "To verify contact us link", "steps": [{"name": "To fetch the website with url: 'http://bestintownpizzas.com/home'", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"c:\\python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\Digital Suppliers\\Desktop\\bestintown\\methods.py\", line 34, in fetching_website\n    raise e\n  File \"C:\\Users\\Digital Suppliers\\Desktop\\bestintown\\methods.py\", line 31, in fetching_website\n    assert driver.title == \"Home | Best in Town Pizza\"\n"}, "attachments": [{"name": "website_screen", "source": "8070fad2-6550-4014-a755-76201b7d80de-attachment.png", "type": "image/png"}], "parameters": [{"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"0882a240a9162ef32ec47129a552d96d\")>"}, {"name": "site_url", "value": "'http://bestintownpizzas.com/home'"}], "start": 1669109682451, "stop": 1669109700385}], "start": 1669109680765, "stop": 1669109700388, "uuid": "5be924ab-a308-4e96-94d6-6ac7f5fc1fb0", "historyId": "2a26da4a76bfd54ea004aa6cc7810e12", "testCaseId": "d695eedd73dde080800b62de22e16250", "fullName": "test_profile_elements.TestBestInTownPizzaProfile#test_03_verify_contactUs", "labels": [{"name": "suite", "value": "test_profile_elements"}, {"name": "subSuite", "value": "TestBestInTownPizzaProfile"}, {"name": "host", "value": "DESKTOP-JHQBC5Q"}, {"name": "thread", "value": "19804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_profile_elements"}]}